INFO - 2023-09-18 23:51:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-18 23:51:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-18 23:52:01 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'useradmin', Object(SensitiveParameterValue), 'upn212evaldoc', 3306, '', 0)
#1 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#7 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('12345678', 'email,password', Array, NULL, 'password')
#8 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array)
#9 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#10 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#11 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#12 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#13 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\public\index.php(67): CodeIgniter\CodeIgniter->run()
#15 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Commands\Server\rewrite.php(47): require_once('E:\\development\\...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\app\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
#6 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Validation\Validation.php(312): App\Validation\Userrules->validateUser('12345678', 'email,password', Array, NULL, 'password')
#7 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array)
#8 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#9 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\app\Controllers\UserController.php(27): CodeIgniter\Controller->validate(Array, Array)
#10 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\CodeIgniter.php(934): App\Controllers\UserController->login()
#11 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#12 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\public\index.php(67): CodeIgniter\CodeIgniter->run()
#14 E:\development\Proyectos\UPN212TEZ\evaldoc_linux\system\Commands\Server\rewrite.php(47): require_once('E:\\development\\...')
#15 {main}
CRITICAL - 2023-09-18 23:52:01 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(10): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(312): App\Validation\Userrules->validateUser('12345678', 'email,password', [...], null, 'password')
 7 SYSTEMPATH\Validation\Validation.php(170): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', [...], [...])
 8 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\UserController.php(27): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserController->login()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\development\\Proyectos\\UPN212TEZ\\evaldoc_linux\\public\\index.php')
INFO - 2023-09-18 23:52:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-18 23:52:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-18 23:52:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-18 23:52:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2023-09-18 23:52:25 --> [DEPRECATED] Function strftime() is deprecated in APPPATH\Controllers\Admin\EvaluacionController.php on line 1428.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin\EvaluacionController->downloadTeacherReport('1')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\EvaluacionController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\development\\Proyectos\\UPN212TEZ\\evaldoc_linux\\public\\index.php')
INFO - 2023-09-18 23:56:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2023-09-18 23:56:16 --> [DEPRECATED] Function strftime() is deprecated in APPPATH\Controllers\Admin\EvaluacionController.php on line 1428.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin\EvaluacionController->downloadTeacherReport('1')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\EvaluacionController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\development\\Proyectos\\UPN212TEZ\\evaldoc_linux\\public\\index.php')
CRITICAL - 2023-09-18 23:56:16 --> Call to undefined method TCPDF::setFooterCallback()
in APPPATH\Controllers\Admin\EvaluacionController.php on line 1605.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin\EvaluacionController->downloadTeacherReport('1')
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\EvaluacionController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\development\\Proyectos\\UPN212TEZ\\evaldoc_linux\\public\\index.php')
